NEW_MODULE_1/NEW_MODULE_1/Program.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+
+namespace NEW_MODULE_1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("N=");
+            int N =Convert.ToInt32(Console.ReadLine());
+            int[] numb_toConvert = new int[N];
+            Random random = new Random();
+            for (int i = 0; i < numb_toConvert.Length; i++)
+            {
+                numb_toConvert[i] = random.Next(0,25);
+            } // creat array
```

Comment:
The code can be separated into the standalone methods (functions), so it would be much easier to read and understand + your beautiful comments would make more sense